sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1654,12 +1690,8 @@ the finalize call either extending the basic command graph proposal
 or layered as a separate extension proposal.
 
 == Non-implemented features
-The following features are not yet or only partially implemented:
+The following features are not yet implemented:
```

Comment:
We don't mention subgraphs here, but we have limitations to what we currently support there, so think we should say something

From our README
> Implemented, with the limitations that a subgraph can only be added as a node to any parent graph once, and will not correctly execute by itself after being added as a sub-graph.

Reply:
done.